src/components/02-components/card-expreciences/card-experiences.tsx

```diff
@@ -0,0 +1,230 @@
+import { Button, Box, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardNumber from '../card-number/card-number';
+
+export type BrandProps = {
```

Comment:
Why `BrandProps`? Use the right name for this type, based on the component name.